docs/CustomUIA.md

```diff
@@ -0,0 +1,125 @@
+<!-- Copyright (c) Microsoft Corporation. All rights reserved.
+     Licensed under the MIT Licence. -->
+
+## Custom UI Automation properties
+Axe Windows supports the inspection of [custom UI Automation properties](https://docs.microsoft.com/en-gb/windows/win32/winauto/uiauto-custompropertieseventscontrolpatterns). The custom properties for which Axe Windows should register are described in a configuration file that is loaded once per session and applies to all inspected UI Automation elements.
+
+### Configuration file format
+Custom UI Automation configuration data is specified in a [JSON](https://en.wikipedia.org/wiki/JSON#Syntax) formatted text file in the configuration directory. The configuration file consists of one object with the following possible attributes:
+
+#### metadata
+This attribute, if specified, can contain values to describe the file as determined by the user. It may be useful, for instance, to store version or authorship information. Example:
+
+```
+  "metadata": {
+    "version": "1.0",
+    "modified": "1970-01-01"
+  }
+```
+
+#### properties
+This attribute contains an array of objects, one per defined custom UI Automation property. Each property object contains the following fields:
+
```

Comment:
What's the best way to format these properties? For now I left them as headings with paragraphs of text. A definition list perhaps? (but I couldn't get it to render in GitHub-flavoured markdown).

Reply:
Maybe use the style in https://github.com/microsoft/axe-windows/blob/main/docs/AutomationReference.md ?